arangod/Cluster/ClusterComm.cpp

```diff
@@ -350,10 +350,13 @@ void ClusterComm::initialize() {
 ////////////////////////////////////////////////////////////////////////////////
 
 void ClusterComm::cleanup() {
+
   if (!_theInstance) {
     return;
   }
 
+  _theInstance->deleteBackgroundThreads();
```

Comment:
it seems unnecessary to call `deleteBackgroundThreads` here, because the dtor of ClusterComm will call it anyway.